exclusion.dic

```diff
@@ -0,0 +1,70 @@
+semver
+Infos
+nupkg
+nuget
+Langpacks
+templateengine
+constantine
+Lexographic
+noops
+balancednesting
+expandvariables
+dotnetcli
+Sdks
+maui
+nuspec
+mytemplate
+myoption
+myname
+othername
+templatecache
+firstrun
+netcoreapp
+netstandard
+doesn
+Installe
```

Comment:
install?

Reply:
There is some auto-generated resx file which line breaks `Installed` in this way.